packages/augur-ui/src/modules/common/labels.tsx

```diff
@@ -550,7 +551,12 @@ export class HoverValueLabel extends React.Component<
 }
 
 export const InvalidLabel = (props: InvalidLabelProps) => {
-  const { text, keyId, openInvalidMarketRulesModal } = props;
+  const {
+    text,
+    keyId,
+    openInvalidMarketRulesModal,
+    tooltipPositioning,
+  } = props;
```

Comment:
we shouldn't bother with this, just do it above instead of `(props: InvalidLabelProps)` we can just do:
```
({
text,
keyId,
openInvalidMarketRulesModal,
tooltipPositioning
}: InvalidLabelProps) => ...
```

Reply:
Got it, will change. Thanks